ubuntu-clock-app/setup/gui/ubuntu-clock-app.desktop

```diff
@@ -132,7 +132,7 @@ Keywords[uk]=time;alarm;alert;clock;world;stopwatch;час;нагадуванн
 Keywords[zh_CN]=time;alarm;alert;clock;world;stopwatch; 时间;闹铃;闹钟;世界时;秒表;
 Keywords[zh_TW]=time;alarm;alert;clock;world;stopwatch;時鐘;鬧鐘;鬧鈴;世界;碼表;
 Exec=ubuntu-clock-app
-Icon=${SNAP}/meta/gui/clock-app@30.png
```

Comment:
No, We need the real path of the icon. This desktop file is read from unity8 outside the SNAP enviroment.